props/saber_BC_buttons.h

```diff
@@ -912,6 +923,14 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
       if (fusor.angle1() >  M_PI / 3) {
         SaberBase::DoEffect(EFFECT_POWERSAVE, 0);
         return true;
+      } else if (fusor.angle1() < - M_PI / 4) {
+      // pointing down
+        sequential_quote_ = !sequential_quote_;
+        beeper.Beep(0.1, 1300);
```

Comment:
Might want to make these beeps different when sequential is turned on vs. off.
Like maybe hi/lo/hi/lo is on and lo/hi/lo is off?
Or maybe use sound library?

Reply:
Good Idea, thanks.
Also updated instructions for this feature.